tests/test_connectbox_static.py

```diff
@@ -70,35 +70,6 @@ def getAdminAuth():
     return requests.auth.HTTPBasicAuth(ADMIN_USER, ADMIN_PASSWORD)
 
 
-class ConnectBoxBasicTestCase(unittest.TestCase):
-
-    def testContentResponseType(self):
-        # URLs under content should return json
-        r = requests.get("%s/content/" % (getTestBaseURL(),))
-        r.raise_for_status()
-        self.assertIsInstance(r.json(), list)
-
-    def testAdminNeedsAuth(self):
```

Comment:
why is this being removed? It's exercising the webserver rather than the interface code, so I'd expect it to stay.